base/base/sort.h

```diff
@@ -30,13 +30,13 @@ void partial_sort(RandomIt first, RandomIt middle, RandomIt last, Compare compar
 template <typename RandomIt, typename Compare>
 void sort(RandomIt first, RandomIt last, Compare compare)
 {
-    ::pdqsort(first, last, compare);
+    ::stdext::bitsetsort(first, last, compare);
 }
 
 template <typename RandomIt>
 void sort(RandomIt first, RandomIt last)
 {
     using value_type = typename std::iterator_traits<RandomIt>::value_type;
     using comparator = std::less<value_type>;
```

Comment:
I highly recommend to randomize the range in sanitizers and debug mode, same for partial_sort and nth_element

Maybe in a separate PR